<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStateView</class>
 <widget class="QWidget" name="DialogStateView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QGroupBox" name="gbStateName">
          <property name="title">
           <string>Name</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leStateName"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KPushButton" name="pbRemoveState">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="pbEditState">
          <property name="text">
           <string>Edit</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="gbRequiredFields">
        <property name="title">
         <string>Required Fields</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KListWidget" name="lwRequiredFields"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KPushButton" name="pbAddRequiredField">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="pbRemoveRequiredField">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="gbTurns">
          <property name="title">
           <string>Turns</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>337</width>
           <height>175</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="KListWidget" name="lwTurns"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KPushButton" name="pbAddTurn">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="pbEditTurn">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="pbRemoveTurn">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pbEditState</sender>
   <signal>clicked(bool)</signal>
   <receiver>pbRemoveRequiredField</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbEditState</sender>
   <signal>clicked(bool)</signal>
   <receiver>pbAddRequiredField</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbEditState</sender>
   <signal>clicked(bool)</signal>
   <receiver>pbRemoveTurn</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbEditState</sender>
   <signal>clicked(bool)</signal>
   <receiver>pbEditTurn</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbEditState</sender>
   <signal>clicked(bool)</signal>
   <receiver>pbAddTurn</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbEditState</sender>
   <signal>clicked(bool)</signal>
   <receiver>pbRemoveState</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbEditState</sender>
   <signal>clicked(bool)</signal>
   <receiver>leStateName</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbEditState</sender>
   <signal>clicked(bool)</signal>
   <receiver>gbStateName</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
